composer.json

```diff
@@ -0,0 +1,5 @@
+{
+    "require": {
```

Comment:
Miss the autoload part (see [symfony-standard configuration](https://github.com/symfony/symfony-standard/blob/2.8/composer.json) for a reference), project name, type, license, and PHP required version.